test/drenv/addons/rook/external_cluster/start.py

```
@@ -0,0 +1,200 @@
+    env["NAMESPACE"] = ROOK_CEPH_NAMESPACE
+    env["KUBECONTEXT"] = managed_cluster
+
+    for line in cluster_details.split("\n"):
+        if line.startswith("export "):
+            _, rest = line.split("export ", 1)
+            k, v = rest.split("=", 1)
+            env[k] = v
+
+    path = _cache.get_file(CACHE_KEY, IMPORT_SCRIPT_URL)
+
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)